rpm/libhybris.spec

```diff
@@ -267,7 +267,6 @@ autoreconf -v -f -i
   %{!?qa_stage_devel:--enable-debug} \
   %{!?qa_stage_devel:--enable-trace} \
   --with-android-headers=/usr/lib/droid-devel/droid-headers \
-  --enable-experimental \
```

Comment:
This would make all other devices that depend on this flag to stop working.

An effort is ongoing to unify libhybris source code that will benefit devices with Mali and those needing Adreno quirks etc: https://github.com/libhybris/libhybris/pull/360